aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSyncClient.java

```diff
@@ -118,10 +118,11 @@ public static AppSyncClient via(@NonNull GraphQLBehavior api,
             }
         };
         final Consumer<ApiException> failureConsumer =
-            failure -> onFailure.accept(new DataStoreException(
-                "Failure performing sync query to AppSync.",
-                failure, AmplifyException.TODO_RECOVERY_SUGGESTION
-            ));
+            failure -> {
+                onFailure.accept(new DataStoreException(
```

Comment:
The Java lambda style guideline usually recommend not using {} for single line.

Reply:
Good catch.